src/css/main.css

```diff
@@ -689,30 +689,39 @@ header .h1-container .subtitle-container img {
   padding: 0 1em;
 }
 .filter-key ul li {
-  display: flex;
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  gap: 10px;
 }
 .filter-key ul li span.color-key {
   height: 15px;
-  width: 25px;
+  width: 20px;
   border-radius: 5px;
 }
 .filter-key ul li span.color-key.one {
-  background-color: #0852C1;
+  background-color: #DE1919;
 }
 .filter-key ul li span.color-key.two {
-  background-color: #8E47F3;
+  background-color:#AD3453;
 }
 .filter-key ul li span.color-key.three {
-  background-color: #D84091;
+  background-color: #7C4F8C;
 }
 .filter-key ul li span.color-key.four {
-  background-color: #EB4F12;
+  background-color: #4B6AC6;
 }
 .filter-key ul li span.color-key.five {
-  background-color: #FFED39;
+  background-color: #1A85FF;
 }
 
+.filter-key h4{
+  text-align: center;
+}
+.percentile-text{
+  text-align: right;
+}
```

Comment:
per above note, if we keep the `flex` over the `grid`, just add a `width: 100%` here to get the `text-align` to work!